File: Praktikum Pemrograman I Modul 4/Praktikum I Modul4  07-11-2023/C/PRAK401-2310817110015-AKHMAD_CHAIDAR_ANANDA.c
#include<stdio.h>

int main(){

    // deklarasi Variabel
    int bil;
    char simbol;

    // input
    scanf("%d %c", &bil, &simbol);

    // deklarasi Variabel
    // perulangan
    int i;
    for(i = 1; i <= 50; i++){

        // pengkondisian
        if (i % bil == 0){
            printf("%c ", simbol);
            continue;
        }

        // output 
        printf("%d ", i);
    }
    
    return 0;
}

File: Praktikum Pemrograman I Modul 4/Praktikum I Modul4  07-11-2023/Python/PRAK401-2310817110015-AKHMAD_CHAIDAR_ANANDA.py
# input
bil, simbol = input().split()

bil = int(bil) # mengkonversi variabel bilangan menjadi tipe data int

# perulangan
for i in range(1, 50 + 1):
    
    # pengkondisian
    if (i % bil == 0): 
        print(simbol, end=' ')
        continue
    
    # output 
    print(f"{i}", end=' ')